transactions/transactions.py

```diff
@@ -201,7 +205,7 @@ async def promote(self, ctx, user: discord.Member, team_name: str):
             await ctx.send("Either {0} isn't on a team right now or his current team can't be found".format(user.name))
 
     @commands.guild_only()
-    @commands.command()
+    @commands.command(aliases=["setTransactionChannel"])
```

Comment:
Just go ahead and rename this method to setTransactionChannel. I don't know why I had tried to shorten it originally but there's no need to.

Reply:
Done. I added the old names as aliases.